inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_reshape.cpp

```diff
@@ -17,15 +17,18 @@
 namespace vpu {
 
 void dynamicToStaticShapeReshape(std::shared_ptr<ngraph::Node> target) {
+    const auto reshape = ngraph::as_type_ptr<ngraph::opset3::Reshape>(target);
+    VPU_THROW_UNLESS(reshape, "dynamicToStaticShapeReshape transformation is not applicable for {}, it should be {} instead",
+                     target, ngraph::opset3::Reshape::type_info);
```

Comment:
Are you sure this will print what you expect? How this print would look like if `target` is not castable to `Reshape`?